tikv-control.md

```diff
@@ -522,6 +522,56 @@ success!
 > - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether the specified `region_id` is validated or not.
 > - You need to run this command for all stores where specified Regions' peers are located.
 
+### Flashback
+
+TiDB v6.4.0 introduces the [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) syntax, You can use it to restore the entire cluster to a specific point in time. For ease of use off TiDB, tikv-ctl provides the `flashback` command since v6.5.3. This command supports Flashback operations at the TiKV level.
+
+> **Note:**
+>
+> - The `flashback` command writes the old data of a specific point in time with the latest timestamp, and will not delete the current data. So before using this feature, you need to ensure that there is enough storage space for the old data and the current data.
+> - This command only supports local mode. It prints `flashback all stores success!` when successfully run.
+
+#### Prerequisites
+
+Before executing the `flashback` command, you need to configure the cluster via `pd-ctl config set halt-scheduling true` command [stop PD scheduling](pd-control.md#config-show--set-option-value--placement-rules).
+
+#### Usage
+
+```shell
+tikv-ctl --pd <pd_address:port> flashback -v <target_timestamp>
+```
+
+Use the `--pd` option to specify the access address of the PD. Use the `-v` option to specify the point in time of the Flashback.
+
+By default, this command will Flashback the entire cluster. If you need to flashback specified Regions or key range, you can use the following options:
+
+- Use the `-r` option to specify the Regions, with multiple Regions separated by `,`.
+- Use `--start` and `--end` to specify all Regions within a key range (no range limit by default, Hex format).
+
+When the command runs successfully, it will print `flashback all stores success!`. You can also view the execution progress via the [Raft admin > Peer in Flashback State](/grafana-tikv-dashboard.md#raft-admin) monitor entry.
```

Comment:
```suggestion
When the command runs successfully, it will print `flashback all stores success!`. You can also view the execution progress via the [Raft admin > Peer in Flashback State](/grafana-tikv-dashboard.md#raft-admin) statistic.
```